.github/workflows/build-test.yml

```diff
@@ -0,0 +1,45 @@
+name: Snyk Vulnerability Scan
+
+on:
+  push:
+    branches:
+      - main 
+  pull_request:
+    branches:
+      - main  
+
+jobs:
+  snyk-scan:
+    runs-on: ubuntu-latest
+
+    steps:
+   
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+   
+      - name: Set up Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Comment:
we should be using at least node 20, preferably 22 but 20 is acceptable right now.

Reply:
Updated to version 20